pinot-minion/src/main/java/com/linkedin/pinot/minion/executor/BaseSegmentConversionExecutor.java

```diff
@@ -113,7 +113,7 @@ public void executeTask(@Nonnull PinotTaskConfig pinotTaskConfig) throws Excepti
       // segment will not be submitted again
       Header ifMatchHeader = new Header(HttpHeaders.IF_MATCH, originalSegmentCrc);
       String minionUserAgentParameter =
-          CommonConstants.Minion.HTTP_TASK_TYPE_HEADER_PREFIX + taskType + "/" + MINION_CONTEXT.getMinionVersion();
+          CommonConstants.Minion.HTTP_TASK_TYPE_HEADER_PREFIX + taskType + CommonConstants.Minion.MINION_HEADER_SEPARATOR + MINION_CONTEXT.getMinionVersion();
```

Comment:
please add this as a method in PinotUserAgentHeader class. Suggest
String constructUserAgentHeader(String taskType) {
}
and you can replace lines 116 through 118 with this method call.